src/selected-tool.php

```diff
@@ -137,19 +137,20 @@
                   <div class="col-xl-12">
                     <div class="row">
                       <?php
-                        $sql2 = $con->prepare("SELECT name, example, command FROM commands WHERE tool=$tool AND type='input' ORDER BY name");
+                        $sql2 = $con->prepare("SELECT name, example, command, value FROM commands WHERE tool=$tool AND type='input' ORDER BY name");
```

Comment:
same comment for the value as explained above. I suggest we keep the same strategy as the other commands, unless we have reasons for change the approach now.

Reply:
The reason I did this was some parameters we want to specify a default value in a case user did not. eg for `hping3` how much packets to send.